common4j/src/main/com/microsoft/identity/common/java/util/JsonUtil.java

```diff
@@ -50,7 +50,8 @@ public static Map<String, String> extractJsonObjectIntoMap(final String jsonStri
         final Map<String, String> responseItems = new HashMap<>();
         while (keyIterator.hasNext()) {
             final String key = keyIterator.next();
-            responseItems.put(key, jsonObject.getString(key));
+            final String value = jsonObject.get(key).toString();
```

Comment:
If jsonObject.get(key) is null, toString()  could throw NPE

Reply:
I'll add a check, good catch!